libs/blocks/media/media.js

```diff
@@ -77,6 +77,10 @@ export default async function init(el) {
       text.classList.add('text');
       decorateAvatar(text);
       decorateBlockText(text, blockTypeSizes[size], blockType);
+      const plainActionArea = text.querySelector('[class*="body-"]:has(a:not(.con-button)):last-child:not(.action-area)');
+      const hasTextNode = plainActionArea
+        && [...plainActionArea.childNodes].some((n) => n.nodeType === 3 && n.textContent.trim() !== '');
```

Comment:
whats nodeType 3? 🤔

Reply:
Should we say n.nodeType === Node.TEXT_NODE? that's more common across the codebase https://github.com/search?q=repo%3Aadobecom%2Fmilo%20TEXT_NODE&type=code

Reply:
I chose this version for brevity. `nodeType === 3` ([here](https://github.com/search?q=repo%3Aadobecom%2Fmilo+%22.nodeType+%3D%3D%3D+3%22&type=code)) is used in 9 files (albeit a lot of them in MEP folders), while `Node.TEXT_NODE` ([here](https://github.com/search?q=repo%3Aadobecom%2Fmilo+%22.nodeType+%3D%3D%3D+Node.TEXT_NODE%22&type=code)) is used in 7 files. I do agree it raises less eyebrows though, so I'll make the change, thanks for noticing!